deps/wazuh_testing/wazuh_testing/qa_ctl/run_tests/Pytest.py

```diff
@@ -74,6 +74,7 @@ def run(self, ansible_inventory_path):
 
         assets_folder = 'assets/'
         assets_zip = "assets.zip"
+        html_report = "/home/vagrant/"
```

Comment:
Be careful with paths ended in `/`. Its really necessary?

Reply:
You cannot hardcode this path, note that this will not only run on machines deployed with vagrant.